groclient/client.py

```diff
@@ -963,11 +963,12 @@ def get_df(
 
         Parameters
         ----------
-            show_revisions : boolean, optional
-                False by default, meaning only the latest value for each period. If true, will return
-                all values for a given period, differentiated by the `reporting_date` field.
-            show_available_date : boolean, optional
-                False by default. If true, will return the available date of each data point.
+            reporting_history : boolean, optional
+                False by default, meaning only the latest value for each period. If true, will return all values for a given period 
+                which have a valid reporting_date.
+            complete_history : boolean, optional
+                False by default, meaning only the latest value for each period. If true, will return all values for a given period, 
+                even if there is no valid reporting_date. Will return values which are differentiated only be available_date.
```

Comment:
@JGrinst 
In the last sentence `... which are differentiated only be available_date.`, I was wonder if it should be 'differentiated only by'? And it's also possible for multiple data points to share the same available date. e.g. when their reporting dates are different.

Reply:
Good point. So perhaps, instead:

"False by default, meaning only the latest value for each period. If true, will return all values for a given period, even if there is not a valid reporting_date."

Reply:
done